doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -61,60 +201,66 @@ To create a Thing for your device:
        }
 
    .. note::
-      The policy example is only intended for development environments.
-      All devices in your production fleet must have credentials with privileges that authorize only intended actions on specific resources.
-      The specific permission policies can vary depending on the use case and should meet business and security requirements.
+      This policy example is only intended for development environments.
+      Make sure to update this to a more restrictive policy before you go into production.
       For more information, refer to the example policies listed in `AWS IoT Core policy examples`_ and `Security best practices in AWS IoT Core`_.
 
-#. Click :guilabel:`Create`.
-#. Go to :guilabel:`Manage` > :guilabel:`All devices`> :guilabel:`Things` and select :guilabel:`Create things`.
-#. Click :guilabel:`Create Thing`.
-#. Enter a name.
-   The default name used by the library is ``my-thing``.
+#. Create the policy using
 
-   .. note::
+   .. code-block:: console
+
+      aws iot create-policy --policy-name my-policy --policy-document file://policy.json
+
+#. Attach the policy to the previously registered certificate using
+
+   .. code-block:: console
 
-      If you are working with an application, that automatically sets the client ID to the IMEI of the device at run-time, you must locate the IMEI of the device and use that as the *Thing name*.
-      If you choose a different name, make sure to :ref:`configure a custom client ID <configuring>` before building.
+      aws iot attach-policy --target <certificate arn> --policy-name my-policy
 
-#. Accept the defaults and continue to the next step.
-#. Select :guilabel:`Auto-generate a new certificate` to generate new certificates.
-#. Select the policy that you created in a previous step.
-#. Click on :guilabel:`Create Thing`.
-#. Download the following certificates and keys for later use:
+   where ``<certificate arn>`` is the ARN of the previously generated device certificate.
 
-   * Thing certificate (:file:`*-certificate.pem.crt`)
-   * The private key (:file:`*.private.pem.key`)
-   * The root CA (choose the Amazon Root CA 1, :file:`AmazonRootCA1.pem`)
-#. Click :guilabel:`Done`.
+.. _creating_a_thing:
 
-The certificates that you created or added for your Thing in AWS IoT must be stored on your device so that it can successfully connect to AWS IoT.
+Creating a thing
+================
 
-.. _flash_certi_device:
+Next, the *Thing* needs to be created in AWS IoT core by performing the following steps:
 
-Provisioning of the certificates
-********************************
+1. Create the *Thing* using
 
-.. include:: /includes/cert-flashing.txt
+   .. code-block:: console
 
-For nRF70 Series devices, the TLS credentials need to be provisioned to the Mbed TLS stack at runtime before establishing a connection to AWS IoT.
```

Comment:
I think we need to keep this section in the library docs. After all, its a library feature.

Reply:
In the tab for the nRF70, I've amended the last step to say `#. Provision the certificates and private key at runtime to the Mbed TLS stack, as shown in the :ref:`aws_iot` sample.` - do you think that is sufficient?